Document that print doesn't work in Python models (#3318)
fishtown-analytics/docs.getdbt.com: website/docs/docs/build/python-models.md

- **Syntax differences** are even more pronounced. Over the years, dbt has done a lot, via dispatch patterns and packages such as `dbt_utils`, to abstract over differences in SQL dialects across popular data warehouses. Python offers a **much** wider field of play. If there are five ways to do something in SQL, there are 500 ways to write it in Python, all with varying performance and adherence to standards. Those options can be overwhelming. As the maintainers of dbt, we will be learning from state-of-the-art projects tackling this problem and sharing guidance as we develop it.
- **These capabilities are very new.** As data warehouses develop new features, we expect them to offer cheaper, faster, and more intuitive mechanisms for deploying Python transformations. **We reserve the right to change the underlying implementation for executing Python models in future releases.** Our commitment to you is around the code in your model `.py` files, following the documented capabilities and guidance we're providing here.

> - **Lack of print() support.** Because your Python model is compiled and run entirely inside the data platform without oversight from dbt, the output of commands like `print()` is not shown in dbt's logs. 

Reviewer: ```suggestion
- **Lack of `print()` support.** The data platform runs and compiles your Python model without dbt's oversight. This means it doesn't display the output of commands such as `print()` in dbt's logs. 
```

> - **Lack of print() support.** Because your Python model is compiled and run entirely inside the data platform without oversight from dbt, the output of commands like `print()` is not shown in dbt's logs. 

Reviewer: sugg active voice

```suggestion
- **Lack of `print()` support.** The data platform runs and compiles your Python model without dbt's oversight. This means it doesn't display the output of commands such as `print()` in dbt's logs. 
```
